src/llvm/lib/Target/MSP430/MSP430ISelLowering.h

```diff
@@ -36,8 +36,8 @@ namespace llvm {
       /// Y = RRC X, rotate right via carry
       RRC,
 
-      /// Clear carry flag
-      CLRC,
+      /// Rotate right via carry, carry is set to zero beforehand
```

Comment:
I would prefer slightly more "sound" name. So one won't confuse with usual shift.

Reply:
`rra` = rotate right arithmetically
`rrl` = rotate right logically which sounds good enough to me
Maybe just a comment change will suffice?

Reply:
But we have side effect with clearing the carry. So if we'd try to start using this we need to take this into account. So I'd at least keep "C" in the name :)

Reply:
Reasonable, will do.